tests/integration/hello_test.py

```diff
@@ -0,0 +1,76 @@
+import json
+import subprocess
+from threading import Thread
+from time import sleep, time
+
+from talker.client import get_talker
+
+
+def run_docker_compose():
+    print('running docker-compose...')
+    subprocess.Popen('docker-compose up', shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    print('docker compose is done')
+
+
+def get_redis_container_ip():
+    while True:
+        redis_container_id = subprocess.check_output(
+            "docker ps | grep redis| awk -F '    ' '{print $1}'",
+            shell=True).strip().decode('utf-8')
+
+        if redis_container_id:
+            inspect_res_str = subprocess.check_output('docker inspect {}'.format(redis_container_id), shell=True)
+            inspect_res = json.loads(inspect_res_str)
+            redis_container_ip = inspect_res[0]['NetworkSettings']['Networks']['integration_default']['IPAddress']
```

Comment:
`docker inspect` allows you to pass a path in the JSON output: https://stackoverflow.com/questions/17157721/how-to-get-a-docker-containers-ip-address-from-the-host